samples/nrf9160/location/src/main.c

```diff
@@ -0,0 +1,290 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <string.h>
+#include <zephyr.h>
+#include <nrf_modem_at.h>
+#include <modem/lte_lc.h>
+#include <modem/location.h>
+#if defined(CONFIG_NRF_CLOUD_PGPS)
+#include <date_time.h>
+#endif /* CONFIG_NRF_CLOUD_PGPS */
+
+static struct loc_config config;
+static struct loc_method_config methods[2];
```

Comment:
I'm thinking if we should have an example with all three methods but they don't nicely fit into any examples.
Adding GNSS to WiFi example doesn't look good because you don't want GNSS to be after WiFi.
And periodic would then need flagging for WiFi.
Maybe this just has to be ignored.

Reply:
if you got wifi enabled in your setup, then it is worth do a wifi trial before any other.

Reply:
Discussed that it isn't a very good example, so won't be added.